### PR TITLE
docs and samples : running on Azure Container Apps

### LINKED ISSUE #1339: [Docs]: App Service / ACA deployment
### What happened?

Missing docs on App Service container deployment.

Also, is ACA supported? - if so, a sample would be helpful

### Version

0.5.34

### What database are you using?

Azure SQL

### What hosting model are you using?

_No response_

### Which API approach are you accessing DAB through?

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by MassimoC**
## Why make this change?

- Closes #1339 

## What is this change?

- update the running-in-azure.md to cover the Azure Container Apps case
- added the Azure Container Apps case to the samples

## How was this tested?

- [x] Integration Tests (running the samples script)

## Sample Request(s)

- the new script (azure-container-apps-deploy.sh) uses the same approach as to the existing azure-deploy.sh. Same naming convention, logging, etc...

### CODE REVIEW

**Comment by MassimoC** [comment]
@microsoft-github-policy-service agree

File: samples/azure/azure-container-apps-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  -g $RESOURCE_GROUP \
+  -f ./bicep/dab-on-aca.bicep \
+  -p appName=$CONTAINERAPPS_APP_NAME dabConfigFileName=$DAB_CONFIG_FILE_NAME mountedStorageName=$FILE_SHARE environmentId=$CONTAINERAPPS_ENVIRONMENTID
+
+echo "get the azure container app FQDN" | tee -a log.txt
+ACA_FQDN=$(az containerapp show -n $CONTAINERAPPS_APP_NAME -g $RESOURCE_GROUP --query properties.configuration.ingress.fqdn -o tsv | tr '[:upper:]' '[:lower:]' | tr -d '[:space:]')
+
+echo "you can now try out the API at the following address : https://${ACA_FQDN}/api/<your-entity-name>"
+
+echo "done" | tee -a log.txt
```
Comment (Aniruddh25) [inline samples/azure/azure-container-apps-deploy.sh:112]:
```suggestion
echo "done" | tee -a log.txt

```

**Review by Aniruddh25 — COMMENTED** [review]
/azp run

**Review by Aniruddh25 — APPROVED** [review]
LGTM (looks good to merge), thank you for your contributions, @MassimoC !!

**Review by abhishekkumams — APPROVED** [review]

**Pull request merged**